### PR TITLE
XRT: Add fedora and mips64 support

### PR DESCRIPTION
**Pull request by AlooGS**

### CODE REVIEW

**Comment by maxzhen** [comment]
ok to test

File: build/build.sh [hunk 1]
```diff
@@ -30,6 +30,10 @@ if [[ $CPU == "aarch64" ]] && [[ $OSDIST == "ubuntu" ]]; then
     fi
 fi
 
+if [[ $OSDIST == "fedora" ]]; then
+    CMAKE=cmake3
+fi
+
```
Comment (stsoe) [inline build/build.sh:36]: Please move to check for other OSDIST line 11

Comment (AlooGS) [reply build/build.sh]: OK，has done it.

**Comment by stsoe** [comment]
Will merge when passed through our pipeline testing.  Thanks again.

**Pull request merged**